portfolio/src/main/webapp/style.css

```diff
@@ -194,3 +194,11 @@ textarea, input, select, label, button {
   font-family: inherit;
   font-size: 1.3em;
 }
+
+.hidden {
+  display: none;
+}
+
+.hidden > .lds-roller {
```

Comment:
Do you need this since 'display: none' hides all children?

Reply:
Are you referring to this selector? ```.hidden > .lds-roller {```

This line shows the loading icon while the page loads the blob form URL. Is there a better setting to use for that?